api/src/main/java/eu/coatrack/api/Proxy.java

```diff
@@ -55,6 +56,10 @@ public Proxy(String id) {
 
     private String configServerPassword;
 
```

Comment:
I would delete the blank lines between the variables. In this case I see no benefit there.

Reply:
Ok, I removed the blank spaces, and just kept a space for the variable id due to the Id annotation, as for me it is easier to read.

Reply:
Imo blank spaces between annotated variables are useful for readability as you already said. 

Additionally it makes sense to group variables in blocks when they belong strongly together and put blank spaces between the blocks.